src/AppBundle/Controller/DonationController.php

```diff
@@ -33,13 +73,27 @@ public function indexAction(Request $request)
     public function informationsAction(Request $request)
     {
         $amount = (float) $request->query->get('montant');
+        $frequency = $request->query->get('frequence') ?? '01';
```

Comment:
Can be simplified to `$frequency = $request->query->get('frequence', '01');`

You should use constants as well for frequency management:

```
$frequency = $request->query->get('frequence', Frequency::MONTHLY);
```